src/components/SearchBar/index.tsx

```diff
@@ -23,7 +23,7 @@ export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
     super(props);
 
     this.state = {
-      text: props.initialText || 'Search...'
+      text: props.initialText || 'search...'
```

Comment:
This needs to be a capital S

Reply:
okay then I messed up later in the code will fix